src/app/components/ManualEntry.svelte

```diff
@@ -0,0 +1,133 @@
+<script lang="ts">
+  /**
+   * Handles the click when the delte button is pressed
+   * TODO
+   */
+  const handleDelete = (): void => {};
```

Comment:
In general, I'd like the names of these callbacks to be more meaningful than `handleX`. I'd rather they specificy _exactly_ what they do, e.g., `deleteRow`.

```suggestion
   * Handles the click when the delete button is pressed
   * TODO
   */
  const handleDelete = (): void => {};
```